src/Writer.h

```diff
@@ -106,7 +106,7 @@ class Writer
   }
 
   std::string fileName, fileAlias, nameWAlias;
-  bool critical, notify, isOpen, firstWrite;
+  bool critical, notify, isOpen = false, firstWrite;
```

Comment:
I am surprised you did not get any compiler error. You cannot initialized the class variable here!

Reply:
Change made as requested.